### PR TITLE
Feature/profile

### PR DESCRIPTION
**Pull request by Ashishgy**
# 

## Type of PR

- [] - feature
- [] - bugfix

## Description

- Added User profile ( empty version ) for both private and public view
- Made this responsive to laptops and mobile both 

## Checklist

- [] - Provided useful description for this PR
- [] - Provided the root cause of this bugfix
- [] - Tested on supported devices

### CODE REVIEW

**Comment by Ashishgy** [comment]
@kbooz @NunoMartins21  Updated the edit profile section for Early and experienced holder

**Comment by fbenfraj** [comment]
@Ashishgy Can you fix the deployment? Not sure but I think if you `pull origin develop` it might do the job

**Comment by Ashishgy** [comment]
okay @fbenfraj I will check and fix this ASAP

File: apps/website/components/templates/private-profile/PrivateProfileTemplate.tsx [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+              gap={1}
+              sx={{
+                mt: 4,
+              }}
+            >
+              <IconButton
+                sx={{
+                  p: 0,
+                }}
+                //onClick={onShare}
```
[the remark this answers is not in the document]
Comment (Ashishgy) [reply apps/website/components/templates/private-profile/PrivateProfileTemplate.tsx:176]: @rafagomes This share functionailty I had hide previously, but will now Push it again soon. 
 Thanks for pointing out this 👍

**Pull request merged**